swagger.yaml

```diff
@@ -35,6 +55,40 @@ responses:
     description: "Failed to process the request due to an internal error"
 
 definitions:
+  CacheTime:
+    type: object
+    required:
+      - id
+      - path
+      - etag
+    properties:
+      id:
+        $ref: "#/definitions/CacheTimeID"
+        description: "MD5 of the path"
+        type: string
+        example: "1b"
```

Comment:
Why are you duplicating the description, type and example here of CacheTimeID?

Reply:
I (think) it's correct now?